doc/developer-documentation/forgae-config/config.md

```diff
@@ -0,0 +1,18 @@
+# forgae-config
+
+## Install 
+
+```
+npm install forgae-utils
+```
+
+## Usage 
+```javascript
+const config = require('forgae-config');
+```
+
+The package is responsible for setting up a healthy local node. 
```

Comment:
missing information about the local compiler that is spawned together with the node